packages/models-library/requirements/_tools.txt

```diff
@@ -8,23 +8,23 @@ appdirs==1.4.4
     # via
     #   black
     #   virtualenv
-black==21.5b1
+black==21.6b0
```

Comment:
If you are bumping black do you need to submit also a formatting commit?

Reply:
that is a good point, we could do this from now on. All black upgrades are followed by a re-formatting
I would propose this time to create a separate PR in which we black-format the entire repo with this version